### ISSUE TITLE
wrong split argument in evaluation when getting loaders

### ISSUE THREAD

**Issue by cod3licious** [atomistic-machine-learning/schnetpack#151]
For spk_run the split argument for train is [n_train, n_val] while for eval it's ["train", "validation", "test"]. When getting the loaders for train/val/test data (line 34), `args` is passed, which works in train mode but in eval mode it's not the number of samples anymore but instead the list with up to 3 fold names (with all 3 that's also one argument to many that is passed to the train_test_split function). Not sure though if it should be replaced by `train_args` instead since then the `batch_size` is not what was specified.

**Comment by Stefaanhess** [comment]
Hey @cod3licious ,
you are right. I added a fix in #152
